api/app/tests/test.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+from django.contrib.auth.models import Permission
+from jsonschema import validate
+from rest_framework.test import APITestCase
+
+from tests.apps.users.factories import SuperUserFactory, UserFactory
+
+
+class SIAReadPermissionMixin:
+    _sia_read = None
+
+    @property
+    def sia_read(self):
+        if self._sia_read is None:
+            self._sia_read = Permission.objects.get(codename='sia_read')
+        return self._sia_read
+
+
+class SIAWritePermissionMixin:
+    _sia_write = None
+
+    @property
+    def sia_write(self):
+        if self._sia_write is None:
+            self._sia_write = Permission.objects.get(codename='sia_write')
+        return self._sia_write
+
+
+class SuperUserMixin:
+    @property
+    def superuser(self):
+        return SuperUserFactory.create()
+
+
+class SimpleUserMixin:
+    @property
+    def user(self):
+        return UserFactory.create(
+            first_name='User',
+            last_name='Simple',
+        )
+
+
+class SIAReadUserMixin(SIAReadPermissionMixin):
+    @property
+    def sia_read_user(self):
+        user = UserFactory.create(
+            first_name='SIA-READ',
+            last_name='User',
+        )
+        user.user_permissions.add(self.sia_read)
+        return user
+
+
+class SIAWriteUserMixin(SIAReadPermissionMixin, SIAWritePermissionMixin):
```

Comment:
SIAReadPermissionMixin doesn't seem to be used.

Reply:
Done